Blog - observability v0.2.0 (#77)
arcus-azure/arcus: community/blog/2020-05-Azure-Dependency-Tracking-in-Observability-v0.2.0/README.md

> # Azure dependency tracking in Observability v0.2.0

Reviewer: How about this title?
```suggestion
# Measure a variety of Azure dependencies with Observability v0.2
```

> With the release of the next version of [Arcus.Observability v0.2.0](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.2.0) the support for dependency tracking of Azure resources is greattly expanded.

Reviewer: ```suggestion
With the release of the next version of [Arcus Observability v0.2](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.2.0) the support for dependency tracking of Azure resources is greatly expanded.
```

> With the release of the next version of [Arcus.Observability v0.2.0](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.2.0) the support for dependency tracking of Azure resources is greattly expanded.

Reviewer: I would add a paragraph before this referring to our previous post

Author: Added a sentance before it.

![iot hub application map](media/iot-hub-application-map.png)

## Azure Blob storage dependency tracking

We allow you to measure Azure Blob storage dependencies using the `LogBlobStorageDependency` overload.
Tracking this depedency, shows on the application map:

![blob storage application map](media/blob-storage-application-map.png)

## Azure Cosmos Db dependency tracking

We allow you to measure Cosmos Db dependencies using the `LogCosmosDdbDependency` overload.
Tracking this dependency, shows on the application map:

![cosmos db application map](media/cosmos-db-application-map.png)

## Azure Event Hubs dependency tracking

We allow you to measure Event Hubs dependencies using the `LogEventHubsDependency` overload.
Tracking this dependency, shows on the application map:

![event hubs application map](media/event-hubs-application-map.png)

## Azure Table storagedependency tracking

We allow you to measure Table storage dependencies using the `LogTableStorageDependency` overload.
Tracking this dependency, shows on the application map:

![table storage application map](media/table-storage-application-map.png)

## SQL dependency tracking

> Tracking SQL dependencies was already available starting from [version v0.1.0](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.1.0), but this version also provides an additional feature to make tracking these kind of dependencies more consumer friendly.

Reviewer: ```suggestion
Tracking SQL dependencies was already available starting from [version v0.1](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.1.0), but this version also provides an additional feature to make tracking these kind of dependencies more consumer friendly.
```

> Instead of providing the server, database, ... separatly, this version allows you to pass along the SQL connection string and extract the required information from it.

Reviewer: ```suggestion
Instead of providing all the connection information by hand, we now allow you to pass the SQL connection string and we'll extract the required information from it.
```

# Measure a variety of Azure dependencies with Observability v0.2

> In a [previous post](https://www.codit.eu/blog/announcing-arcus-observability/) we announced the Arcus Observability library as a way to empower your applications with a variety of observability essentials.

Reviewer: Adding a few examples
```suggestion
In a [previous post](https://www.codit.eu/blog/announcing-arcus-observability/) we announced the Arcus Observability library as a way to empower your applications with a variety of observability essentials such as correlation, empowering telemetry and dependency tracking.
```

> With the release of the next version of [Arcus Observability v0.2](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.2.0) the support for dependency tracking of Azure resources is greatly expanded.

Reviewer: Adding a new line for readability
```suggestion

With the release of [Arcus Observability v0.2](https://github.com/arcus-azure/arcus.observability/releases/tag/v0.2.0) our support for dependency tracking of Azure resources is greatly expanded.
```
